api/build.gradle

```diff
@@ -72,9 +71,9 @@ def versions = [
         sonarPitest     : '0.5',
         lombok          : '1.18.32',
         pact_version    : '4.1.7',
-        reformS2sClient : '3.1.4',
-        serenity        : '2.6.0',
-        serenityCucumber: '3.0.1',
+        reformS2sClient : '5.2.0',
+        serenity        : '4.2.3',
+        serenityCucumber: '4.2.3',
```

Comment:
I believe it shouldn't break anything if you bump this to the latest version, which is 4.2.9